frontend/views/containers/modals/SignUp.vue

```diff
@@ -3,190 +3,42 @@
     template(slot='title')
       i18n Sign Up
 
-    form(
-      novalidate
-      ref='form'
-      name='formData'
-      data-test='signup'
-      @submit.prevent='signup'
-    )
-      label.field
-        i18n.label Username
-        input.input#name(
-          :class='{error: $v.form.name.$error}'
-          name='name'
-          @input='debounceName'
-          ref='username'
-          data-test='signName'
-          v-error:name='{ tag: "p", attrs: { "data-test": "badUsername" } }'
-        )
-
-      label.field
-        i18n.label Email
-        input.input#email(
-          :class='{error: $v.form.email.$error}'
-          name='email'
-          v-model='$v.form.email.$model'
-          type='email'
-          data-test='signEmail'
-          v-error:email='{ tag: "p", attrs: { "data-test": "badEmail" } }'
-        )
-
-      form-password(
-        :label='L("Password")'
-        :value='form'
-        :v='$v.form'
-        @input='(newPassword) => {password = newPassword}'
-      )
-
-      p.error(v-if='ephemeral.errorMsg') {{ ephemeral.errorMsg }}
-
-      .buttons.is-centered
-        i18n.is-primary.is-centered(
-          tag='button'
-          type='submit'
-          :disabled='$v.form.$invalid'
-          data-test='signSubmit'
-        ) Create account
+    form-signup(@submitSucceeded='submit')
 
     template(slot='footer')
       p
-        i18n Already have an account?&nbsp;
-        a.link(@click='showLoginModal' data-test='goToLogin')
-          i18n Login
+        i18n Already have an account?
+        | &nbsp;
+        i18n.link(tag='a' @click='showLoginModal' data-test='goToLogin') Log in
 </template>
 
 <script>
-import { required, minLength, email } from 'vuelidate/lib/validators'
-import { validationMixin } from 'vuelidate'
-import { debounce } from '@utils/giLodash.js'
 import { REPLACE_MODAL } from '@utils/events.js'
 import sbp from '~/shared/sbp.js'
-import { nonWhitespace } from '@views/utils/validators.js'
 import ModalTemplate from '@components/Modal/ModalTemplate.vue'
-import FormPassword from '@components/Forms/Password.vue'
-import L from '@view-utils/translations.js'
+import FormSignup from '@containers/forms/FormSignup.vue'
 
-// TODO: fix all this
 export default {
-  name: 'SignUpModal',
-  mixins: [validationMixin],
+  name: 'Signup',
```

Comment:
Please keep it `SignUp` or rename the file to `Signup.vue`. I personally think it should be kept as `SignUp`

Reply:
I changed to signup because we say login instead of logIn. So it was to keep consistency. I'll rename the file to `Signup.vue`